x-pack/plugins/apm/public/components/app/Correlations/correlations_table.tsx

```diff
@@ -29,22 +33,27 @@ interface Props<T> {
   status: FETCH_STATUS;
   cardinalityColumnName: string;
   setSelectedSignificantTerm: (term: T | null) => void;
+  onFilter: () => void;
 }
 
-export function SignificantTermsTable<T extends SignificantTerm>({
+export function CorrelationsTable<T extends SignificantTerm>({
   significantTerms,
   status,
   cardinalityColumnName,
   setSelectedSignificantTerm,
+  onFilter,
 }: Props<T>) {
   const history = useHistory();
   const columns: Array<EuiBasicTableColumn<T>> = [
     {
       width: '100px',
       field: 'score',
-      name: 'Score',
+      name: i18n.translate(
+        'xpack.apm.correlations.correlationsTable.scoreLabel',
+        { defaultMessage: 'Score' }
+      ),
       render: (_: any, term: T) => {
-        return <EuiCode>{Math.round(term.score)}</EuiCode>;
+        return <EuiBadge>{Math.round(term.score)}</EuiBadge>;
```

Comment:
I think we talked about showing the score visually - similar to how impact is displayed on the transactions table. I think impact should be calculated as:
```
score / maxScoreInCurrentSet
```

Reply:
Actually, another approach I think is better (it seems arbitrary but I think it'll yield better results):
```
const max = Math.max(maxScore, medianScore * 2);
const impact = score / max;
```

Try it out an see what works best.